extensions/cornerstone/src/commandsModule.ts

```diff
@@ -580,6 +580,23 @@ function commandsModule({
     storePresentation: ({ viewportId }) => {
       cornerstoneViewportService.storePresentation({ viewportId });
     },
+
+    attachProtocolViewportDataListener: ({ protocol, stageIndex }) => {
+      const EVENT = cornerstoneViewportService.EVENTS.VIEWPORT_DATA_CHANGED;
+      const command = protocol.callbacks.onViewportDataInitialized;
+      const numPanes = protocol.stages?.[stageIndex]?.viewports.length;
```

Comment:
Consider changing this to...

`const numPanes = protocol.stages?.[stageIndex]?.viewports?.length ?? 0;`